Documentation/LCDLanguageFont.md

```diff
@@ -38,6 +38,7 @@ We have two different technologies for the displays:
   * de    German
   * es    Spanish
   * ru    Russian
+  * bgr   Bulgarian
```

Comment:
The ISO 639-1 code for Bulgarian is just __bg__. http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes